src/screens/map/DimensionScreen.js

```diff
@@ -88,27 +92,24 @@ const DimensionScreen = props => {
 
   return (
     <View style={styles.container}>
+      <Navbar>
+        <Confirm
+          id='unit-screen-confirm'
+          question={t('unitScreen.abort.question')}
+          approveText={t('unitScreen.abort.abort')}
+          denyText={t('unitScreen.abort.continue')}
+          onApprove={() => props.navigation.navigate('Home')}
+          icon='arrow-left'
+          tts={false}
+          style={{
+            borderRadius: 2,
+            borderWidth: 1,
+            borderColor: Colors.dark
+          }}
+        />
+        <ProfileButton onPress={() => props.navigation.navigate('Profile')} />
+      </Navbar>
       {renderDimensions()}
-
-      <View style={styles.navigationButtons}>
-        <View style={styles.routeButtonContainer}>
-          <RouteButton
-            onlyIcon
-            icon='arrow-alt-circle-left' handleScreen={() => {
-              props.navigation.navigate('Map')
-            }}
-          />
-        </View>
-
-        <View style={styles.routeButtonContainer}>
-          <RouteButton
-            onlyIcon
-            icon='arrow-alt-circle-right' handleScreen={() => {
-              props.navigation.navigate('Unit')
-            }}
-          />
-        </View>
-      </View>
     </View>
```

Comment:
No confirm on dimension screen required, just a button to navigate back to the map (not to home).